README.md

```diff
@@ -597,6 +671,47 @@ end
 
 Nested facets have the same options of json facets
 
+#### BlockJoin Json Facet
+
+You can use `json_facet` on children or parents, based on the data
+you are querying on.
+
+##### `on_child` operator
+
+Use this operator when searching on parent documents.
+Faceting is performed on child documents related to parents found
+in the query.
+
+```ruby
+# Search for all books with specified title and facets
+# on the timestamp of reviews.
+# An additional filter on children can be specified inside the on_child operator. 
+Sunspot.search(Book) do
+  fulltext 'awesome book title', fields: [:title]
+  
+  json_facet :review_date, block_join: (on_child(Review) do
```

Comment:
this block nesting looks 😕 
Instead of yielding, can we use a lambda/proc passed explicitly as an argument?